Adyen/adyen-magento
# Update apple_pay.phtml

**Description**
do not display apple pay on shopping cart if there is no product in it.

**Tested scenarios**
if you webstore allow you to show empty shopping cart page
- go to shopping cart page
- see an exception.

**Fixed issue**:  #888
